routes.py

```diff
@@ -2,34 +2,33 @@
 import binascii
 import sys
 import hashlib
-from flask import Flask, jsonify, request, render_template
+import ast
+import requests
+from flask import Flask, jsonify, request, render_template, session, redirect, url_for
 from blockchain import Blockchain
 from uuid import uuid4
 from levelpy import leveldb
-from ecdsa import SigningKey, NIST384p
+from ecdsa import SigningKey, VerifyingKey, NIST384p
 from apscheduler.schedulers.background import BackgroundScheduler
 
 # Instantiate our Node
 app = Flask(__name__)
+app.secret_key = b'z*\x82\xa2\x1c\x8cT\x92\x0e\x0c@\x16\x06\xe2?\x9c'
 scheduler = BackgroundScheduler()
 
 # Generate a globally unique address for this node
 node_identifier = str(uuid4()).replace('-', '')
-host = sys.argv[1]
+ip = sys.argv[1]
 port = sys.argv[2]
+host = ip+":"+port
 
 # Instantiate the Blockchain
 blockchain = Blockchain()
-blockchain.register_node(host+':'+port)
+blockchain.register_node('192.168.0.7:5006')
 
 # Create levelDB
-db = leveldb.LevelDB('./db/'+port, create_if_missing=True)
-accountdb = leveldb.LevelDB('./db/account', create_if_missing=True)
-
-# Initialize
-privkey = pubkey = b''
-wallet = ''
-amount = -1
+db = leveldb.LevelDB('./db/'+host, create_if_missing=True)
```

Comment:
It would better if this program makes the required directory automatically before this code.